reference/5.1/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -29,6 +29,77 @@ the parameters in a command. Splatting is valid on simple and advanced
 functions. For more information, see [about_Functions](about_Functions.md) and
 [about_Splatting](about_Splatting.md).
 
+## Type conversion of parameter values
+
+When you supply strings as arguments to parameters that expect a different
+type, PowerShell implicitly converts the strings to the parameter target type.
+Advanced functions perform culture-invariant parsing of parameter values.
+
+By contrast, a culture-sensitive conversion is performed during parameter
+binding for compiled cmdlets.
+
+In this example, we create a cmdlet and a script function that take a
+`[datetime]` parameter. The current culture is changed to use German settings.
+A German-formatted date is passed to the parameter.
+
+```powershell
+# Create a cmdlet that accepts a [datetime] argument.
+Add-Type @'
+  using System;
+  using System.Management.Automation;
+  [Cmdlet("Get", "Date_Cmdlet")]
+  public class GetFooCmdlet : Cmdlet {
+
+    [Parameter(Position=0)]
+    public DateTime Date { get; set; }
+
+    protected override void ProcessRecord() {
+      WriteObject(Date);
+    }
+  }
+'@ -PassThru | % Assembly | Import-Module
+
+[cultureinfo]::CurrentCulture = 'de-DE'
+$dateStr = '19-06-2018'
+
+Get-Date_Cmdlet $dateStr
+```
+
+```Output
+Dienstag, 19. Juni 2018 00:00:00
+```
+
+As shown above, cmdlets use culture-sensitive parsing to convert the string.
+
+```powershell
+# Define an equivalent function.
+function Get-Date_Func {
+  param(
+    [DateTime] $Date
+  )
+  process {
+    $Date
+  }
+}
+
+[cultureinfo]::CurrentCulture = 'de-DE'
+
+# This German-format date string doesn't work with the invariant culture.
+# E.g., [datetime] '19-06-2018' breaks.
+$dateStr = '19-06-2018'
+
+Get-Date_Func $dateStr
+```
+
+Advanced functions use culture-invariant parsing, which results in the
+following error.
+
+```Output
+Get-Date_Func: Cannot process argument transformation on parameter 'Date'.
+Cannot convert value "19-06-2018" to type "System.DateTime". Error: "String
+'19-06-2018' was not recognized as a valid DateTime."
```

Comment:
On 5.1, I only see `Error: "String was not recognized as a valid DateTime."`.  I don't see the actual string in that part of the output. 6 and up show it.